### DISCUSSION TITLE
CIR of multiple BS

### DISCUSSION THREAD

**Discussion by Advaith2212** [NVlabs/sionna#244] [Q&A] [answered]
In case of a scenario, where a UE has a channel between more than one BS, how do I visualise `CIR` ? 

Is the delay of the second/third antenna normalised with respect to the LoS of the first antenna ?

**Comment by jhoydis** [comment, marked as answer]
Hi,

Do you refer to Sionna RT? In this case, you can set ``path.normalize_delays=False`` to get the absolute delays. I am not sure what you mean by visualise? You can simply plot the CIR between any TX-RX antenna pair.
